cli/mqtt.py

```diff
@@ -22,8 +26,8 @@ def mqtt_open(config, insecure):
             acct.mqtt_username,
             acct.mqtt_password,
             printer.mqtt_key,
-            ca_certs="examples/ankermake-mqtt.crt",
-            verify=not insecure,
+            ca_certs=cert_path,
+            verify=insecure,
```

Comment:
Could you walk me through what this line is doing? 

`mqtt_open()` is called twice
1. cli
2. web/service

When called as part of web/service, then the `insecure` is set to `True` (line 14)
When called as part of the cli it is set to `env.insecure` which is defaulting to `env.insecure = insecure` on line 79 of the main() function.

I'll probably defer this to @chrivers on this one because I'm not sure why the 2 levels of indirection for this boolean.

Reply:
Ok, so I found the error and discovered my err. I assumed we were setting `insecure` incorrectly here but it was actually in the webserver that we were doing so. I have reverted this change here and set up the webserver to allow passing the insecure flag through to the app configuration for flask.